.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please revert the prettier-related changes, these should not be in this PR.

Reply:
There is not package lock of the dependencies. Feel free to amand my PR. As having the write access on the repository, you have write access to my fork's branch.

Reply:
Also I expect changing any part of the package.json invalidating travis cache and making all consecutives build red.

Reply:
Not sure I understand what you mean. Do you say you had to change the prettier configuration because the prettier version wasn't fixed in the package, and prettier updated their defaults in a new version?

If that's the case, you should just remove the `~` in the prettier version in the `package.json`.